src/test/java/fi/otavanopisto/kuntaapi/test/ManagementIncidentMocker.java

```diff
@@ -0,0 +1,103 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.containing;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.junit.Assert.fail;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.tomakehurst.wiremock.matching.StringValuePattern;
+
+import fi.metatavu.management.client.model.Incident;
+
+@SuppressWarnings ({"squid:S1166", "squid:S1075"})
+public class ManagementIncidentMocker extends AbstractBaseMocker {
+
+  private static final String PATH_TEMPLATE = "%s/%s";
+  private static final String PAGES_PATH = "/wp-json/wp/v2/incident";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1075)